### PR TITLE
Create Profile RohithJN.json

### PR DESCRIPTION
**Pull request by Rohith-JN**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2064"></a>

### CODE REVIEW

File: public/data/RohithJN.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "url": "https://github.com/Rohith-JN",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/RohithNambiar4",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on Linkedin",
```
Comment (kumarsonsoff3) [inline public/data/RohithJN.json:18]: I recommend capitalizing `i` here in `LinkedIn`. What do you think!? 
```suggestion
        "name": "Connect with me on LinkedIn",
```

File: public/data/RohithJN.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        "icon": "Linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started software development",
+        "date": "December 2019",
+        "icon": "globe",
+        "color": "blue",
+        "description": "Started software development (Python, Javascript, Flutter)",
```
Comment (Panquesito7) [inline public/data/RohithJN.json:34]: I recommend capitalizing the `s` here. What do you think? 🙂
```suggestion
        "description": "Started software development (Python, JavaScript, Flutter)",
```

**Review by imran1509 — COMMENTED** [review]
Looks great to me

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

---
**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**